Content.Client/Strip/StrippingMenu.cs

```diff
@@ -59,8 +59,19 @@ public void AddButton(string title, string name, Action<BaseButton.ButtonEventAr
                     button,
                 }
             });
+        }
+
+        public float GetRequiredHeight()
+        {
+            float sum = 0;
+            for(int i = 0; i < _vboxContainer.ChildCount; i++)
+            {
+                _vboxContainer.GetChild(i).Measure((5000, 5000));
 
+                sum += _vboxContainer.GetChild(i).DesiredSize.Y + _vboxContainer.SeparationOverride ?? 0;
+            }
```

Comment:
you should be able to just measure the vbox directly (which under the hood more or less does the same thing)

Reply:
You are absolutely correct. I had tried measuring the _strippingMenu but hadn't thought of just doing the _vboxContainer. Thanks for the tip!